Makefile

```diff
@@ -14,6 +14,7 @@
 #
 # ESPRUINO_1V3=1          # Espruino board rev 1.3 and rev 1v4
 # PICO_1V3=1              # Espruino Pico board rev 1.3
+# PUCKJS=1                # Puck.js board
```

Comment:
I wouldn't bother with that - it'd be best to add something to the Makefile that actually displayed some helpful text if the bootloader file wasn't found.